config/src/test/java/no/sysco/middleware/kafka/interceptor/config/ConfigHarvesterInterceptorIT.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2018 Sysco Middleware AS.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package no.sysco.middleware.kafka.interceptor.config;
+
+import java.util.Properties;
+import java.util.concurrent.Future;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.testcontainers.containers.KafkaContainer;
+
+/**
+ *
+ * @author 100tsa
+ */
+public class ConfigHarvesterInterceptorIT {

```

Comment:
This is great! Maybe we are trying to achieve the same as [JMH](http://openjdk.java.net/projects/code-tools/jmh/)?

Reply:
We can try at least, I never used JMH from unit tests before, as official docs recommends to do such things separately and use standalone project for JMH benchmarks so they do not disturb each other (at least it was so for 1-2 years ago)

Reply:
Of course, we can take it separately. I've created #5 to track this.